config/crd/bases/metal3.io_provisionings.yaml

```diff
@@ -51,7 +51,7 @@ spec:
                 - Disabled
                 type: string
               provisioningNetworkCIDR:
-                description: ProvisioningNetworkCIDR is the network on which the baremetal nodes are provisioned. The provisioningIP and the IPs in the dhcpRange all come from within this network.
+                description: ProvisioningNetworkCIDR is the network on which the baremetal nodes are provisioned. The provisioningIP and the IPs in the dhcpRange all come from within this network. IPv6 networks cannot be larger than a /64.
```

Comment:
Do you think it could be worth to add the same motivation introduced in the PR description also here (`dnsmasq`) for the sake of clarity?

Reply:
It seems unnecessary to add this as I'm not sure users should care, but I'm happy to update it if you think it's appropriate.